stream_srtp.go

```diff
@@ -116,13 +116,13 @@ type WriteStreamSRTP struct {
 }
 
 // WriteRTP encrypts a RTP header and its payload to the nextConn
-func (w *WriteStreamSRTP) WriteRTP(header *rtp.Header, payload []byte) (int, error) {
-	headerRaw, err := header.Marshal()
+func (w *WriteStreamSRTP) WriteRTP(packet *rtp.Packet) (int, error) {
```

Comment:
This API is on purpose, all of the `pion/` transports do the same thing.

Reply:
I'm a little too eager to make breaking changes for no reason in this case. 

However, it means I'm going to have to use `rtp.MarshalSize()` and `rtp.MarshalTo()` to optimize this code, because it's doing an extra allocation for no reason.